tests/disaster-recovery/mdr/test_app_failover_and_relocate.py

```diff
@@ -0,0 +1,226 @@
+                back to primary cluster once it recovers
+
+        This test is also compatible to be run from UI,
+        pass the yaml conf/ocsci/dr_ui.yaml to trigger it.
+        """
+
+        if config.RUN.get("mdr_failover_via_ui"):
+            ocs_version = version.get_semantic_ocs_version_from_config()
+            if ocs_version <= version.VERSION_4_12:
+                logger.error(
+                    "ODF/ACM version isn't supported for Failover/Relocate operation"
+                )
+                raise NotImplementedError
+
+        acm_obj = AcmAddClusters()
+        workload = dr_workload(num_of_subscription=1)[0]
+
+        # Create application on Primary managed cluster
+        set_current_primary_cluster_context(workload.workload_namespace)
+        primary_cluster_index = config.cur_index
+        node_objs = get_node_objs()
+        primary_cluster_name = get_current_primary_cluster_name(
+            namespace=workload.workload_namespace
+        )
+        self.drcluster_name = primary_cluster_name
+        self.namespace = workload.workload_namespace
+
+        # Stop primary cluster nodes
+        if primary_cluster_down:
+            logger.info("Stopping primary cluster nodes")
+            nodes_multicluster[primary_cluster_index].stop_nodes(node_objs)
+
+        # Fenced the primary managed cluster
+        enable_fence(drcluster_name=self.drcluster_name)
+
+        # Application Failover to Secondary managed cluster
+        secondary_cluster_name = get_current_secondary_cluster_name(
+            workload.workload_namespace
+        )
+        if config.RUN.get("mdr_failover_via_ui"):
+            logger.info("Start the process of Failover from ACM UI")
+            config.switch_acm_ctx()
+            failover_relocate_ui(
+                acm_obj,
+                workload_to_move=f"{workload.workload_name}-1",
+                policy_name=workload.dr_policy_name,
+                failover_or_preferred_cluster=secondary_cluster_name,
+            )
+        else:
+            failover(
+                failover_cluster=secondary_cluster_name,
+                namespace=workload.workload_namespace,
+            )
+
+        # Verify application are running in other managedcluster
+        # And not in previous cluster
+        set_current_primary_cluster_context(workload.workload_namespace)
+        wait_for_all_resources_creation(
+            workload.workload_pvc_count,
+            workload.workload_pod_count,
+            workload.workload_namespace,
+        )
+
+        # Verify the failover status from UI
+        if config.RUN.get("mdr_relocate_via_ui"):
+            config.switch_acm_ctx()
+            verify_failover_relocate_status_ui(acm_obj)
+
+        # Start nodes if cluster is down
+        wait_time = 60
+        if primary_cluster_down:
+            logger.info(
+                f"Waiting for {wait_time} minutes before starting nodes of previous primary cluster"
```

Comment:
is wait_time in seconds or minutes? If seconds then please change "... minutes before starting...."

Reply:
oops, it is in seconds. Will change accordingly